aldjemy/orm.py

```diff
@@ -91,7 +93,7 @@ def _extract_model_attrs(model, sa_models):
 def prepare_models():
     tables = get_tables()
     models = get_django_models()
-    sa_models = getattr(Cache, 'models', {})
+    sa_models = {}
```

Comment:
I vote for the first variant, plus it seems reasonable to start deprecation process for the models  attribute, as its current state is hardly predictable in case of inheritance.